docs/user/manage_facility.rst

```diff
@@ -0,0 +1,15 @@
+.. _manage_facility:
+
+Manage Facility
+~~~~~~~~~~~~~~~~
+
+You can edit facility configuration settings in Kolibri from the **Facility** tab in your **Manage** dashboard.
+
+#. Activate the options you want to make available for the users of your facility.
+#. Click **Save changes** to apply and finish.
+
+.. image:: img/manage_facility.png
```

Comment:
Not sure if the ones in the screenshot are intended to be there soon, or if the current one is the one we want to push.